bin/build_utils_gitlab

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash -e
+
+if [[ "${TRACE}" ]]; then
```

Comment:
This needs to be compared to something I think otherwise it's going to trigger if TRACE has any value:
```
$ TRACE=false
$  if [[ "${TRACE}" ]]; then echo "set"; else echo "not set"; fi
set

$ TRACE=0
$  if [[ "${TRACE}" ]]; then echo "set"; else echo "not set"; fi
set
```